package.json

```diff
@@ -17,15 +14,21 @@
   },
   "homepage": "https://github.com/Wikia/app",
   "devDependencies": {
+    "eslint": "4.19.1",
+    "eslint-plugin-es5": "^1.3.1",
     "istanbul": "^0.4.5",
     "jasmine-core": "^2.8",
+    "jscs-jsdoc": "^2.0.0",
     "karma": "^2.0.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.1",
     "karma-jasmine": "^1.1.1",
     "karma-junit-reporter": "^1.2.0",
     "karma-requirejs": "^1.1.0",
-    "requirejs": "^2.3.5",
-    "jscs-jsdoc": "^2.0.0"
+    "requirejs": "^2.3.5"
+  },
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint ./extensions/wikia/AdEngine/js",
```

Comment:
I don't think you need to specify the full path to `eslint` here. `npm` will resolve it - `eslint ./extensions/wikia/AdEngine/js` is enough here.

Reply:
You're right. Thanks

https://github.com/Wikia/app/pull/15388